server/config/config.go

```diff
@@ -780,6 +780,10 @@ type ScheduleConfig struct {
 
 	// EnableWitness is the option to enable using witness
 	EnableWitness bool `toml:"enable-witness" json:"enable-witness,string"`
+
+	// SlowStoreEvictingAffectedStoreRatioThreshold is the affected ratio threshold when judging a store is slow
+	// A store's slowness must affected more than `store-count * SlowStoreEvictingAffectedStoreRatioThreshold` to trigger evicting.
```

Comment:
```suggestion
	// A store's slowness must exceed `store-count * SlowStoreEvictingAffectedStoreRatioThreshold` stores in the cluster to trigger evicting.
```

Reply:
Addressed

Reply:
Reversed, I think the original one maybe more accurate, for the reason as the below one comment.